opm/parser/eclipse/EclipseState/Tables/VFPProdTable.cpp

```diff
@@ -376,6 +377,11 @@ void VFPProdTable::check() {
                     for (size_type t=0; t<m_data.shape()[0]; ++t) {
                         if (m_data[t][w][g][a][f] < bhp_last) {
                             ++num_decreasing;
+                            points += "At point (FLOW, THP, WFR, GFR, ALQ) = "
```

Comment:
Will not the numbers here be squashed together and unreadable since there is no space between the to_string-generated strings?